TESTING.md

```diff
@@ -269,7 +269,7 @@ WORKING_BRANCH="develop"
 git checkout "${WORKING_BRANCH}"
 ```
 
-Next, build Bottlerocket images and repos and sync TUF repos. 
+Next, build Bottlerocket images and repos and sync TUF repos.
```

Comment:
I wonder if there's an opportunity here to have a linter that will fail if we have hanging whitespace. Is that something the MDlinter does @stmcginnis ?

Reply:
Yep, this is one of the things that would catch.

I wouldn't want it added to this PR since there's already so much, but I think these changes would get us most of the way there to be able to enable markdown linting like in https://github.com/bottlerocket-os/bottlerocket-test-system/pull/514

Then we can make sure these kinds of things don't slip back in.